src/js/app.js

```diff
@@ -718,6 +718,8 @@ requirejs(['jquery','app/storage','knockout','knockout-secure-binding','hjls','a
       _activateTab(tabActivated);
     };
 
+    const enableSend = () => true
```

Comment:
is this used ?

Reply:
I am sorry, I was into other things that I totally forgot it. I will remove it right now, and once I have time I will also try to understand why it's behaving this way. Meanwhile, if you come across any resources that can help me to better understand, please share it.